### DISCUSSION TITLE
What Happened To The Old Owner?

### DISCUSSION THREAD

**Discussion by Darkestangel69** [KevinBytesTheDust/CVAmp#291] [Q&A] [answered]
What happened to jlplenio?

**Comment by johnxdifu** [comment]
I was going to ask the same thing

**Comment by jlplenio** [comment, marked as answer]
Hello everyone, jlplenio here, all is well. I've passed the torch to Kevin (@KevinBytesTheDust) due to increasing workload and to ensure a reasonable work-life balance. He is a fellow coder friend of mine who has helped me with the transition from Selenium to Playwright in the past. The renaming to Crude Viewer Amplifier, or CVAmp for short, is to better reflect the expanded multi-platform support we achieved over the last few years.

**Comment by m4od** [comment]
jlplenio appreciate you man. Been a long time user, thank you <3
